plt-scheduler/src/block_state/blob_store.rs

```diff
@@ -0,0 +1,92 @@
+// Temporarily allow dead_code until we have block state implemented.
```

Comment:
This file is more or less copy-pasted from `wasm-chain-integration`.
Should probably be removed and reused from there instead.